devel/glasskube/files/patch-Makefile.diff

```diff
@@ -0,0 +1,13 @@
+--- Makefile	2024-08-15 14:27:49
++++ Makefile	2024-08-26 10:40:31
+@@ -111,8 +111,8 @@
+ 	$(GOCMD) build -o $(OUT_DIR)/cert-manager ./cmd/cert-manager/
+ 
+ .PHONY: build-cli
+-build-cli: fmt lint-go lint-web web ## Build cli binary.
+-	$(GOCMD) build -o $(OUT_DIR)/glasskube ./cmd/glasskube/
++build-cli: fmt web ## Build cli binary.
++	$(GOCMD) build -ldflags "-X github.com/glasskube/glasskube/internal/config.Version=0.17.0" -o $(OUT_DIR)/glasskube ./cmd/glasskube/
```

Comment:
> `..."-X github.com/glasskube/glasskube/internal/config.Version=0.17.0"`

Hard-coding the version like this right into the Makefile patch will prove annoying as that will need to be updated every time the port gets updated.

It's probably best to add a VERSION variable to the Makefile and do something like:

`$(GOCMD) build -ldflags "-X github.com/glasskube/glasskube/internal/config.Version=$(VERSION)" ...`

Then VERSION can be passed to make using `build.args-append` or `build.pre_args-append`.

Reply:
I took a stab at it.